src/angular-business/chip/chip/chip.component.ts

```diff
@@ -19,24 +29,34 @@ export class ChipComponent {
   }
   private _disabled = false;
 
+  /** Emits the current chip instance on dismissal. */
   @Output()
   dismissed: EventEmitter<ChipComponent> = new EventEmitter();
 
-  @HostBinding('attr.aria-hidden') ariaHidden: null | boolean | 'true';
+  /**
+   * TODO: Refactor to _ariaHidden
+   * @deprecated Internal detail only
+   */
+  ariaHidden: null | boolean | 'true';
```

Comment:
Wouldn't it be better to already prefix it with "_" and provide a getter to ariaHidden?